client/packages/system/src/Stock/Components/InventoryAdjustment/AdjustmentStats.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  useTranslation,
+  Box,
+  Typography,
+  SimpleStatistic,
+  UNDEFINED_STRING_VALUE,
+} from '@openmsupply-client/common';
+import { StockLineRowFragment } from '../../api';
+
+export const ItemDetailAndStats = ({
+  stockLine,
+  variation,
+}: {
+  stockLine: StockLineRowFragment;
+  variation: number;
+}) => {
+  const t = useTranslation();
+
+  const {
+    item: { code, name },
+    packSize,
+    totalNumberOfPacks,
+    availableNumberOfPacks,
+  } = stockLine;
+  return (
+    <Box
+      sx={{
+        borderWidth: 4,
+        borderRadius: '16px',
+        borderStyle: 'solid',
+        borderColor: 'border',
+        padding: 1,
+        margin: '0 auto',
+        width: '520px',
+      }}
+    >
+      <Box sx={{ paddingX: 1, marginBottom: 1 }}>
+        <Typography>
+          {t('label.code')}:
+          <Typography component="span" color="gray.dark">
+            {` ${code} | `}
+          </Typography>
+          {t('label.pack-size')}:
+          <Typography component="span" color="gray.dark">
+            {` ${packSize}`}
+          </Typography>
+        </Typography>
```

Comment:
Any particular reason for the two separate `Typography` blocks inside? When I do this, it looks exactly the same as far as I can tell:
```
<Typography>
          {t('label.code')}:
          <Typography component="span" color="gray.dark">
            {` ${code} | `}
            {t('label.pack-size')}:
            {` ${packSize}`}
          </Typography>
        </Typography>
```

Reply:
In fact, why not just:
```tsx
        <Typography>
          {t('label.code')}:
          <Typography component="span" color="gray.dark">
            {` ${code} | ${t('label.pack-size')}:  ${packSize}`}
          </Typography>
        </Typography>
```